documentation/docs/20-core-concepts/30-form-actions.md

```diff
@@ -344,12 +344,12 @@ To customise the behaviour, you can provide a `SubmitFunction` that runs immedia
 ```svelte
 <form
 	method="POST"
-	use:enhance={({ formElement, formData, action, cancel, submitter }) => {
-		// `formElement` is this `<form>` element
-		// `formData` is its `FormData` object that's about to be submitted
+	use:enhance={({ form, data, action, cancel, controller }) => {
```

Comment:
`form` and `data` were deprecated in #9902 and will be removed in the next major version.